BearlyHumans/squirrelville
# Humans throw out food

This Pull request adds the following functionality to humans
* Add ability for Humans to take out X amount of food from the player when caught
* Has human collect each piece of food individually 
* Has humans detect if they have taken food or not
* Ability to add bins to the home point 
* Humans can walk to bins to deposit food

TODO:
* Still need to add animation for each part

Steps to test part 1 (all done in Lachie Scene):
* Don't grab food
* Run in front of Human
* Check to see if frozen 
* Wait to check in unfrozen
* Try to get caught again

https://user-images.githubusercontent.com/20939519/133597313-e5be68d3-678a-4752-b2db-7e94c4e40844.mp4

Steps to test part 2 (all done in Lachie Scene):
* Grab some food (any amount for different tests)
* Run in front of human
* Check to see if human "picks up" each piece of food
* Wait to see what human does
* Half the time will Stand still for 10 secs (This isn't a bug, its pause waiting for Eating animation)
* Half the time will walk to a bin and wait for 10 secs (This isn't a bug, it pause waiting for deposit animation)
* Try to get caught again

https://user-images.githubusercontent.com/20939519/133598286-d0b1e4e5-3508-404e-abbc-6b9ee738d633.mp4

Trello:
https://trello.com/c/L2tbRxMN
